pkgs/games/openra/builds.nix

```diff
@@ -0,0 +1,33 @@
+pkgs:
+
+let
+  f = import ./common.nix;
+  common = f (builtins.intersectAttrs (builtins.functionArgs f) pkgs // {
+    lua = pkgs.lua5_1;
+    inherit (pkgs.gnome3) zenity;
+  });
+
+  buildOpenRASet = f: args: builtins.mapAttrs (name: value: if builtins.isFunction value then value name else value) (f ({
```

Comment:
Document what `f` and `args` are.

Reply:
I have documented `common.nix` to explain that and more.